Not working on Android Oreo

Hi, guys.
I have a RN app using react-native-notifications and react-native-navigation.
RN is 0.56.
On Android 8, this error appears:
```
07-24 17:03:48.441 2823-2823/? E/AndroidRuntime: FATAL EXCEPTION: main
	Process: chat.rocket.reactnative.debug, PID: 2823
	java.lang.RuntimeException: Unable to start receiver com.google.android.gms.gcm.GcmReceiver: java.lang.IllegalStateException: Not allowed to start service Intent { act=com.google.android.c2dm.intent.RECEIVE flg=0x1000010 pkg=chat.rocket.reactnative.debug cmp=chat.rocket.reactnative.debug/com.wix.reactnativenotifications.gcm.GcmMessageHandlerService (has extras) }: app is in background uid UidRecord{982ece1 u0a86 RCVR idle procs:1 seq(0,0,0)}
		at android.app.ActivityThread.handleReceiver(ActivityThread.java:3259)
		at android.app.ActivityThread.-wrap17(Unknown Source:0)
		at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1677)
		at android.os.Handler.dispatchMessage(Handler.java:105)
		at android.os.Looper.loop(Looper.java:164)
		at android.app.ActivityThread.main(ActivityThread.java:6541)
		at java.lang.reflect.Method.invoke(Native Method)
		at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
		at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:767)
	Caused by: java.lang.IllegalStateException: Not allowed to start service Intent { act=com.google.android.c2dm.intent.RECEIVE flg=0x1000010 pkg=chat.rocket.reactnative.debug cmp=chat.rocket.reactnative.debug/com.wix.reactnativenotifications.gcm.GcmMessageHandlerService (has extras) }: app is in background uid UidRecord{982ece1 u0a86 RCVR idle procs:1 seq(0,0,0)}
		at android.app.ContextImpl.startServiceCommon(ContextImpl.java:1505)
		at android.app.ContextImpl.startService(ContextImpl.java:1461)
		at android.content.ContextWrapper.startService(ContextWrapper.java:644)
		at android.content.ContextWrapper.startService(ContextWrapper.java:644)
		at android.support.v4.content.WakefulBroadcastReceiver.startWakefulService(WakefulBroadcastReceiver.java:99)
		at com.google.android.gms.gcm.GcmReceiver.doStartService(Unknown Source:22)
		at com.google.android.gms.gcm.GcmReceiver.zzd(Unknown Source:0)
		at com.google.android.gms.gcm.GcmReceiver.onReceive(Unknown Source:98)
		at android.app.ActivityThread.handleReceiver(ActivityThread.java:3252)
```

I tried to fix it by setting channelId and finding opened PRs here and on another libs, but without success.
Can somebody help me?

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -1,12 +1,12 @@
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "25"
+    compileSdkVersion 27
+    buildToolsVersion "27.0.3"
 
     defaultConfig {
-        minSdkVersion 16
-        targetSdkVersion 23
+        minSdkVersion 19
+        targetSdkVersion 27
         versionCode 1
         versionName "1.0"
     }
@@ -20,7 +20,7 @@ android {
 
 dependencies {
     // Google's GCM.
-    compile 'com.google.android.gms:play-services-gcm:10.0.1'
+    compile 'com.google.android.gms:play-services-gcm:15.0.1'
 
     compile 'com.facebook.react:react-native:+'
 

--- a/android/src/main/java/com/wix/reactnativenotifications/core/notification/PushNotification.java
+++ b/android/src/main/java/com/wix/reactnativenotifications/core/notification/PushNotification.java
@@ -1,10 +1,12 @@
 package com.wix.reactnativenotifications.core.notification;
 
 import android.app.Notification;
+import android.app.NotificationChannel;
 import android.app.NotificationManager;
 import android.app.PendingIntent;
 import android.content.Context;
 import android.content.Intent;
+import android.os.Build;
 import android.os.Bundle;
 
 import com.facebook.react.bridge.ReactContext;
@@ -136,13 +138,28 @@ public class PushNotification implements IPushNotification {
     }
 
     protected Notification.Builder getNotificationBuilder(PendingIntent intent) {
-        return new Notification.Builder(mContext)
+
+        String CHANNEL_ID = "channel_01";
+        String CHANNEL_NAME = "Channel Name";
+
+        final Notification.Builder notification = new Notification.Builder(mContext)
                 .setContentTitle(mNotificationProps.getTitle())
                 .setContentText(mNotificationProps.getBody())
                 .setSmallIcon(mContext.getApplicationInfo().icon)
                 .setContentIntent(intent)
                 .setDefaults(Notification.DEFAULT_ALL)
                 .setAutoCancel(true);
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            NotificationChannel channel = new NotificationChannel(CHANNEL_ID,
+                                                                  CHANNEL_NAME,
+                                                                  NotificationManager.IMPORTANCE_DEFAULT);
+            final NotificationManager notificationManager = (NotificationManager) mContext.getSystemService(Context.NOTIFICATION_SERVICE);
+            notificationManager.createNotificationChannel(channel);
+            notification.setChannelId(CHANNEL_ID);
+        }
+        
+        return notification;
     }
 
     protected int postNotification(Notification notification, Integer notificationId) {

